lib/editors/jetbrains-ide.js

```diff
@@ -0,0 +1,17 @@
+var settings = {
+  pattern: '{filename} --line {line}'
+};
+
+var detect = function(ide) {
+  return require('../detect').lazy(ide.name, ['code'], '', {
```

Comment:
Second argument is terminal command. If there are no common commands for editor left it empty.